source/hassio/installation.markdown

```diff
@@ -61,13 +61,20 @@ This is the list of packages you need to have available on your system that will
  - network-manager
  - socat
  - software-properties-common
-
+ - docker*
+ 
+*As apt-get might install an old docker version, run the following command as root to install docker:
+```bash
+$ curl -sSL https://get.docker.com | sh
```

Comment:
I doubt that this will work. 1. You want to install something and you are not root. 2. The chance is very high that it breaks an existing Docker installation. 3. You need to update Docker manually in the future.